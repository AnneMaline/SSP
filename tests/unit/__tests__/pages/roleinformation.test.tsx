import { render, screen } from "@testing-library/react";
import RoleInformationPage from "../../../../src/app/Role-information/page";

describe("Role-information Page", () => {
  it("renders correctly", () => {
    const { container } = render(<RoleInformationPage />);
    expect(container).toMatchSnapshot();
  });
});
