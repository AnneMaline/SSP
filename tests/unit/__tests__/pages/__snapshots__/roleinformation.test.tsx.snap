// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Role-information Page renders correctly 1`] = `
<div>
  <div>
    <h1>
      Information about the role
    </h1>
  </div>
</div>
`;
